src/particle/converters/bimap.py

```diff
@@ -85,7 +85,7 @@ def __init__(
         elif isinstance(filename, HasOpen):
             file_object = filename.open()
         else:
-            file_object = open(filename)  # type: ignore[arg-type]
+            file_object = open(filename, encoding="utf_8")  # type: ignore[arg-type]
```

Comment:
Default encoding should always be specified, will be changed to be always unicode in Python 3.15.

Reply:
We're good for a while but being explicit is hardly ever bad 👍.